microsoft/testsuites/vm_extensions/runtime_extensions/common.py

```diff
@@ -5,7 +5,7 @@
 
 from assertpy import assert_that
 from azure.core.exceptions import ResourceExistsError
-from azure.storage.blob import BlobType
+from azure.storage.blob._models import BlobType
```

Comment:
It may not be allowed to reference `_models`. Please try if it's fixed by upgrading storage packages.